src/Microsoft.ML.Core/Data/KeyType.cs

```diff
@@ -59,6 +59,13 @@ public static bool IsValidDataType(Type type)
         /// </summary>
         public ulong Count { get; }
 
+        /// <summary>
+        /// Determine if a DataViewType object is equal to another KeyType instance.
```

Comment:
>DataViewType [](start = 27, length = 12)

We might prefer `<see` tags. Not that I think we'll rename them this late in the game, but the links would probably be nice.